source/Halibut.Tests/Support/SerilogLoggerBuilder.cs

```diff
@@ -14,58 +14,69 @@ namespace Halibut.Tests.Support
     public class SerilogLoggerBuilder
     {
         static readonly ILogger Logger;
+        static readonly ConcurrentDictionary<string, TraceLogFileLogger> TraceLoggers = new();
+
+        TraceLogFileLogger? traceFileLogger;
 
         static SerilogLoggerBuilder()
         {
-            var outputTemplate = 
-                "{TestName} "
+            const string teamCityOutputTemplate =
+                "{TestHash} "
                 + "{Timestamp:HH:mm:ss.fff zzz} "
                 + "{ShortContext} "
                 + "{Message}{NewLine}{Exception}";
 
+            const string localOutputTemplate =
```

Comment:
I noticed we have lost the `TestName` for when it is run locally, as we share this with the `TraceLogsForFailedTestsSink`.

Is that what we want?

Reply:
I'm not 100% I'm interpreting your comment correctly, but I assume you mean that we've lost `TeamName` because the `localOutputTemplate` const doesn't contain it?

The `TestName` field was already empty when running locally, so the logger just didn't print anything out for that field.

Having separate template strings was just an attempt to remove that extra unnecessary space from the start of the log message when we know for certain that the name/hash field is empty.

Let me know if I've misinterpreted and we can talk about it further.

Reply:
Ah, I see. Sorted 😁